.travis.yml

```diff
@@ -23,8 +23,12 @@ matrix:
       sudo: required
     - os: linux
       dist: trusty
-      sudo: required
+      sudo: false
       env: RUN_UI_TESTS=1 SKIP_NOMAD_TESTS=1
+    - os: linux
+      dist: trusty
+      sudo: false
```

Comment:
Huh, how does this work when `before_install` includes a `sudo` command?

Reply:
It is running sudo in a container versus the host 🤷‍♂️